sdk/core/Azure.Core/src/Serialization/SchemaValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
For the SR betas, let's move this into `Azure.Core.Experimental` so that we don't rush the Core types to GA.

Reply:
Moved.